vhdbuilder/packer/install-dependencies.sh

```diff
@@ -381,23 +381,14 @@ done
 # kubelet and kubectl
 # need to cover previously supported version for VMAS scale up scenario
 K8S_VERSIONS="
-1.17.0
-1.16.6
-1.16.4
-1.16.1
-1.14.6_v0.0.5
```

Comment:
not really needed for cluster/agentpool creation, kept for old vmas scale up optimization